.gitignore

```diff
@@ -6,5 +6,6 @@
 /src/docs/doxygen/html/
 /src/docs/doxygen/xml/
 /src/docs/sphinx/_build/
+/src/docs/_build
```

Comment:
I added this because I found a neat vscode plugin that does inline previews, but it builds the docs here.

Reply:
I found a toggle for this, but it's still probably a good thing to keep in case others build directly with sphinx.